flow/cmd/handler.go

```diff
@@ -211,6 +211,36 @@ func (h *FlowRequestHandler) CreateQRepFlow(
 	}, nil
 }
 
+func (h *FlowRequestHandler) dropFlow(
+	ctx context.Context,
+	flowJobName string,
+	deleteStats bool,
+) error {
+	logs := slog.Group("shutdown-log", slog.String(string(shared.FlowNameKey), flowJobName))
+
+	dropFlowWorkflowID := fmt.Sprintf("%s-dropflow-%s", flowJobName, uuid.New())
+	workflowOptions := client.StartWorkflowOptions{
+		ID:                    dropFlowWorkflowID,
+		TaskQueue:             h.peerflowTaskQueueID,
+		TypedSearchAttributes: shared.NewSearchAttributes(flowJobName),
+	}
+
+	if dropFlowHandle, err := h.temporalClient.ExecuteWorkflow(ctx, workflowOptions, peerflow.DropFlowWorkflow, &protos.DropFlowInput{
+		FlowJobName:           flowJobName,
+		DropFlowStats:         deleteStats,
+		FlowConnectionConfigs: nil,
+		SkipDestinationDrop:   true,
```

Comment:
A bit confusing to have SkipDestinationDrop but not SkipSourceDrop. Could do very explicit with both true, or let FlowConnectionConfigs nil do the job

Reply:
May want to still pass FlowConnectionConfigs so that `DropFlowStats` gets handled